apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/RevertFinalDecisionDialog.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+
+import { fakePoolCandidates } from "@gc-digital-talent/fake-data";
+import { OverlayOrDialogDecorator } from "@gc-digital-talent/storybook-helpers";
+import { PoolCandidateStatus } from "@gc-digital-talent/graphql";
+
+import RevertFinalDecisionDialog from "./RevertFinalDecisionDialog";
+
+export default {
+  component: RevertFinalDecisionDialog,
+  title: "Components/Revert final decision dialog",
+  decorators: [OverlayOrDialogDecorator],
+  args: {
+    defaultOpen: true,
+  },
+} as Meta;
+
+const fakedCandidate = fakePoolCandidates(1)[0];
+
+const Template: StoryFn<typeof RevertFinalDecisionDialog> = (args) => (
+  <RevertFinalDecisionDialog {...args} />
+);
+
+const defaultArgs = {
+  id: fakedCandidate.id,
+  status: fakedCandidate.status,
+  expiryDate: fakedCandidate.expiryDate,
+  finalDecisionAt: fakedCandidate.finalDecisionAt,
+};
+
+export const Qualified = Template.bind({});
+Qualified.args = {
+  revertFinalDecisionQuery: {
+    " $fragmentRefs": {
+      RevertFinalDecisionDialogFragment: {
+        ...defaultArgs,
+        status: PoolCandidateStatus.QualifiedAvailable,
+      },
+    },
```

Comment:
Instead of doing this, it is easier if you use the `makeFragmentData` helper. Example:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/c500f21d289e6a374fd7b6945fdc0f2e1f833d82/apps/web/src/pages/Classifications/UpdateClassifictionForm.stories.tsx#L13-L17

Reply:
Nice fixed by [ec2f9e0](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10152/commits/ec2f9e040593ea6e0f17b4784d7c8e6d3e7c93e6)